ci-operator/config/openstack-k8s-operators/nova-operator/openstack-k8s-operators-nova-operator-master.yaml

```diff
@@ -43,6 +51,16 @@ tests:
   commands: make gofmt || true
   container:
     from: src
+- as: precommit-check
+  commands: |
+    export SKIP=no-commit-to-branch
+    export GOFLAGS=
+    export PRE_COMMIT_HOME=/tmp
```

Comment:
I think we need `export GOLANGCI_LINT_CACHE=/tmp` as well to make golangci-lint happy with in this CI container.

Reply:
thanks, added

Reply:
Are you sure you added it? I don't see it.